site/gatsby-site/src/components/visualizations/Timeline.js

```diff
@@ -93,23 +91,50 @@ const DataPoint = ({ bucket, groupRadius, radius, yScale }) => {
         </>
       ) : (
         <circle
-          className={`${bucket[0].isOccurrence ? 'fill-danger' : 'fill-gray-900'}`}
+          className={`${
+            bucket[0].isOccurrence
+              ? 'fill-danger'
+              : bucket[0].isResponse
+              ? 'fill-green-500'
+              : 'fill-gray-900'
+          }`}
           cy={0}
           r={radius}
         />
       )}
 
       {bucket[0].isOccurrence ? (
-        <text dominantBaseline="middle" className="text-[14px]" dx={16}>
+        <text
+          dominantBaseline="middle"
+          className={`text-[14px] ${bucket[0].isResponse ? 'fill-green-700' : ''}`}
+          dx={16}
+        >
           {bucket[0].title}
+          {bucket[0].isResponse && (
+            <>
+              {' '}
+              - <Trans>Response</Trans>
+            </>
+          )}
         </text>
       ) : (
         <a href={bucket[0].mongodb_id ? `#r${bucket[0].report_number}` : ''}>
-          <text dominantBaseline="middle" className="text-[14px]" dx={16}>
+          <text
+            dominantBaseline="middle"
+            className={`text-[14px] ${bucket[0].isResponse ? 'fill-green-700' : ''}`}
+            dx={16}
+          >
             {bucket[0].title}
+            {bucket[0].isResponse && (
+              <>
+                {' '}
+                - <Trans>Response</Trans>
+              </>
+            )}
           </text>
         </a>
       )}
+      {bucket[0].isResponse && <Badge color={'success'}>RESPONSE_TAG</Badge>}
```

Comment:
@clari182 Why do we have the text `RESPONSE_TAG` here?
<img width="1416" alt="image" src="https://user-images.githubusercontent.com/6564809/203854713-f09142c1-5d93-462d-bfe4-caff2398e724.png">

Reply:
Removed!